### DISCUSSION TITLE
[Feature Request] Confirm Actions

### DISCUSSION THREAD

**Discussion by castdrian** [community/community#145491] [Mobile]
I've submitted quite a few issues on accident without having finished the issue body or title by accidentally hitting submit. A confirmation alert using UIAlert would be good.

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.186.0 (146974874)
Device: iPhone 15 Pro Max (iOS 17.4)
</details>

**Comment by iskae** [comment]
Hi @castdrian 👋 

That is an interesting point 🤔 We generally treat deletion as a destructive action and show a confirmation dialog for them, but creation was never treated as a destructive action 😅 

Let me reach out to our UX experts and see if we can improve things 🙇

**Reply by castdrian** [reply]
That'd be lovely, I am quite embarrassed by a few accidental unfinished issues that have ended up in discord webhooks so that's on the internet forever now 🥲
